client/js/table-view.js

```diff
@@ -158,10 +158,10 @@ class TableView {
     const col = this.currentCellLocation.col
     const row = this.currentCellLocation.row
 
+    // checks to see if in col header when user clicks add row
+    // adds column to the right of the current column
+    // I should have implemented in table-model
     if (row === -1){
-      // echo, !DRY
-      // shifts column
-      // would like to put this funtion into table-model "shiftColumn"
       for(var c = this.model.numCols - 1; c > col; c--) {
```

Comment:
Replace "var" with either "const" or "let" throughout. Use "let" instead of "var" in for loops.